### PR TITLE
Updates project documentation

### PR DESCRIPTION
**Pull request by chentex**
Signed-off-by: Vicente Zepeda Mas <vzepedamas@suse.com>

Adds documentation to the project.
Fixes the scope of the option `--ignore-errors`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+### Issues ###
+
+If you have found a bug in this project or have a question, first make sure
+that the issue you are facing has not already been reported by another user. If
+the issue you are facing has already been reported and you have more
+information to provide, feel free to add a follow-up comment (but avoid adding
+"me too" style comments as it distracts from discussion). If you couldn't find
+an existing report for your issue, feel free to [open a new issue][issue-new].
+If you do not wish to use proprietary software to submit an issue, you may send
+an email to <mailto:vzepedamas@suse.de> and I will submit an issue on your behalf.
```
Comment (flavio) [inline CONTRIBUTING.md]: Use this email address: `SUSE Containers Team <containers@suse.com>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  happen versus what actually happened.
+
+[issue-new]: https://github.com/openSUSE/helm-mirror/issues/new
+[xy-problem]: http://xyproblem.info/
+
+### Submitting Changes ###
+
+In order to submit a change, you may [create a pull request][pr-new].  If you
+do not wish to use proprietary software to submit an pull request, you may send
+an email to <mailto:vzepedamas@suse.de> and I will submit a pull request on your
```
Comment (flavio) [inline CONTRIBUTING.md]: Same here

File: doc/man/mirror-help.1.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+% mirror-help(1) # mirror help - Displays the usage statement
+% Vicente Zepeda
```
Comment (flavio) [inline doc/man/mirror-help.1.md]: SUSE LLC

File: doc/man/mirror-inspect-images.1.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+% mirror-inspect-images(1) # mirror inspect-images - Extract container images used by charts
+% Vicente Zepeda
```
Comment (flavio) [inline doc/man/mirror-inspect-images.1.md]: SUSE LLC

File: doc/man/mirror-version.1.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+% mirror-version(1) # Version - show program current version
+% Vicente Zepeda
```
Comment (flavio) [inline doc/man/mirror-version.1.md]: SUSE LLC

File: doc/man/mirror.1.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
+% mirror(1) # mirror - Mirror chart repositories
+% Vicente Zepeda
```
Comment (flavio) [inline doc/man/mirror.1.md]: SUSE LLC

**Comment by chentex** [comment]
@flavio the `ignore-erros` was already implemented but I had it as a global flag and it didn't make sense since it is specific to the `inspect-images` sub-command. While writing the documentation I noted it and I moved it, the implementation stays the same.

**Pull request merged**